selenium-system-test/src/test/scala/hts/steps/UserDetailsSteps.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package hts.steps
+
+import cucumber.api.DataTable
+import hts.pages.{AuthorityWizardPage, Page}
+import hts.utils.Configuration
+
+import scala.collection.JavaConverters.mapAsScalaMapConverter
+
+class UserDetailsSteps extends Steps{
+
+  var name: Option[String] = None
+  var nino: Option[String] = None
+  var dateOfBirth: Option[String] = None
+  var email: Option[String] = None
+
+  Given("""^an applicant has the following details:$"""){ (applicantDetails:DataTable) =>
+    val expectedData = applicantDetails.asMaps(classOf[String], classOf[String])
```

Comment:
this returns a java `List` instead of a scala `List`. Working with a scala `List` will allow you to do a `foreach` instead of the `for` loop you have below